pkg/agent/openflow/pipeline_windows.go

```diff
@@ -115,13 +115,16 @@ func (c *client) uplinkSNATFlows(localSubnet net.IPNet, category cookie.Category
 // snatImplementationFlows installs flows that implement SNAT with OVS NAT.
 func (c *client) snatImplementationFlows(nodeIP net.IP, category cookie.Category) []binding.Flow {
 	snatIPRange := &binding.IPRange{StartIP: nodeIP, EndIP: nodeIP}
-	l3FwdTable := c.pipeline[l3ForwardingTable]
-	nextTable := l3FwdTable.GetNext()
+	snatTable := c.pipeline[snatTable]
+	// Do not decrement TTL on the SNATed packet on Windows. This is because the SNATed packet is output to antrea-gw0
+	// first to learn the destination MAC using the host ARP cache. And antrea-gw0 will decrement the TTL when it
+	// forwards the packet to the bridge interface on which the host IP address is configured.
+	nextTable := c.pipeline[l3DecTTLTable].GetNext()
```

Comment:
The snatTable's default next table is l2ForwardingCalcTable, why we don't use its next table directly here but uses a non-related table's?
I think the meaning of default table is packets will go to it by default unless we explicitly change it for some reason. So I don't quite understand what's the indication here.

Reply:
I want to explicitly skip "l3DecTTLTable" for the reason I commented here.

Reply:
I understand the comment but what's the point of the default next table then? I think this would confuse readers that snatTable's next table is `l3DecTTLTable` so we explicitly skip it. Overriding nextTable only makes sense when we use a non-default value. If the normal path should go snatTable -> l3DecTTLTable, it would make sense to skip it like this. When the pipeline is designed as snatTable-> l2ForwardingCalcTable, doesn't it imply we only delTTL for particular traffic explicitly? I'm fine with the comment but want to avoid this confusing nextTable declaration.

Reply:
Sure, updated.